libs/libm/newlib/include/_newlib_version.h

```diff
@@ -0,0 +1,33 @@
+/****************************************************************************
+ * libs/libm/newlib/include/_newlib_version.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __LIBS_LIBM_NEWLIB_INCLUDE__NEWLIB_VERSION_H
+#define __LIBS_LIBM_NEWLIB_INCLUDE__NEWLIB_VERSION_H
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define _NEWLIB_VERSION "4.3.0"
+#define __NEWLIB__ 4
+#define __NEWLIB_MINOR__ 3
+#define __NEWLIB_PATCHLEVEL__ 0
```

Comment:
Maybe we can `-D` this from `libs/libm/newlib/Make.defs`? Or this file is needed inside the newly code?